src/main/java/org/fundacion/jala/converter/service/ExtractText.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.service;
+
+import net.sourceforge.tess4j.Tesseract;
+import net.sourceforge.tess4j.TesseractException;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+
+public class ExtractText {
+
+    private String language;
+    private String pathFile;
+    private String nameOutputFile;
+
+    public ExtractText(final String language, final String pathFile) {
+        this.language = language;
+        this.pathFile = pathFile;
+    }
+
+    public ExtractText(final String language, final String pathFile, final String nameOutputFile) {
+        this.language = language;
+        this.pathFile = pathFile;
+        this.nameOutputFile = nameOutputFile;
+    }
+
+    /**
+     *
+     * @return a string with the value of the selected language
+     */
+    public String getLanguage() {
+        return language;
+    }
+
+    /**
+     *
+     * @return a string with the path of the images
+     */
+    public String getPathFile() {
+        return pathFile;
+    }
+
+    /**
+     *
+     * @return a string with the name of file generate
+     */
+    public String getNameOutputFile() {
+        return nameOutputFile;
+    }
+
+    /**
+     *  extract text for a image
+     */
+    public void extractText() {
```

Comment:
You Could add loggers to this class instead of print in console, also add loggers to the try catch funtions

Reply:
sorry I forgot I already put them

Reply:
Ready Jorge, could you check it please

Reply:
As a suggestion you can use this: LOGGER.error("Execute Tesseract Exception" + e.getLocalizedMesagge); to log the especific error in the logs file

Reply:
thansk jorge, I will put it